lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -11,6 +11,10 @@ class Base
           ::RubyParser::SyntaxError,
         ]
 
+        DEFAULT_MASS_THRESHOLD = 28
+        BASE_POINTS = 1_500_000
+        POINTS_PER = 50_000
```

Comment:
That sounds okay to me.  Only arguments to consider against:

1) The `per` naming is consistent with some of our usages in engines and classic:
classic: [cost_per](https://github.com/codeclimate/analyzer/blob/master/lib/quality/rubric.rb#L5)
eslint: [cost_per](https://github.com/codeclimate/codeclimate-eslint/blob/master/lib/checks.js#L120)
rubocop: [per_violation_points](https://github.com/codeclimate/codeclimate-rubocop/blob/master/lib/cc/engine/violation_decorator.rb#L33)

2) the total amount of `overage points` is really this number times the overage (mass - threshold).